lib/AST/Macro.cpp

```diff
@@ -484,7 +485,7 @@ MacroKind Macro::Kind(void) const noexcept {
   }
 }
 
-std::string Macro::KindName(void) const noexcept {
+std::string_view Macro::KindName(void) const noexcept {
   const static std::string KindNames[] = {
```

Comment:
These can be `const status std::string_view`, and then move them out of the function body, into an anonymous namespace.